perception/autoware_bytetrack/lib/include/lapjv.h

```diff
@@ -66,6 +66,8 @@
 
 #if 0
 #include <assert.h>
+
+#include <cstdio>
```

Comment:
I think we can remove this change because we will be covering it by another PR: https://github.com/autowarefoundation/autoware.universe/pull/9563/files

Reply:
I'm still going to approve it because there will be no harm. (You will just have a conflict in the above PR)

Reply:
Good catch!

I'll remove it now.

Reply:
Reverted with https://github.com/autowarefoundation/autoware.universe/pull/9569/commits/2b170200c51c60969f530dd545174c4709a64c98